deploy/deploy_citus_server.yml

```diff
@@ -10,6 +10,8 @@
     - {role: log-courier, when: "install_logstash"}
     - role: jdk
     - role: postgresql-server
+    - role: citus
+    - role: postgresql-server-config
   tasks:
```

Comment:
Could you make this `post_tasks` instead?  I'm not sure that it really matters, but it may help to be explicit to ensure no future regressions in the execution order.